src/khoj/processor/speech/text_to_speech.py

```diff
@@ -0,0 +1,51 @@
+import json  # Used for working with JSON data
+import os
+
+import requests  # Used for making HTTP requests
+from bs4 import BeautifulSoup
+from markdown_it import MarkdownIt
+
+# Define constants for the script
+CHUNK_SIZE = 1024  # Size of chunks to read/write at a time
+XI_API_KEY = os.getenv("ELEVEN_LABS_API_KEY", None)  # Your API key for authentication
```

Comment:
Variable name maybe confusion. Maybe just call it ELEVEN_API_KEY (similar to ELEVEN_API_URL) or rename ELEVEN_API_URL to XI_API_URL